tests/testthat/test-pipe_return_linter.R

```diff
@@ -0,0 +1,47 @@
+test_that("pipe_return_linter skips allowed usages", {
```

Comment:
Maybe use `trim_some()` instead of passing a vector of lines of code?

Reply:
I was editing in nano :sweat_smile: 

much easier from VScode